### ISSUE TITLE
A11y_AzureDataStudio_BigDataClusterOverviewPage_ScreenReader :Missing tab structure for the controls under the "Cluster Details".

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7666]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;DesktopApp;#NVDA;#JAWS;#SQL Azure Data Studio;#Win32

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : NVDA , JAWS

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
8. "Big Data Cluster Overview page " should open.
9. Start NVDA.
10. Navigate till "Cluster Details " using arrow navigation .
11. Observe the announcement by screen reader .

**Actual:**
The "SQL Servers ","HDFS", "Sparks", "Control", "Gateway" and "App" controls which are visually appearing as tab structure but are not structured as tabs.
Similar for "Basic data cluster Overview"
![image](https://user-images.githubusercontent.com/54656883/66652904-6f496600-ec54-11e9-8fd3-0b34b732fece.png)

**Similar issue is also observed with JAWS .**
**Similar issue is also observed with the following screens:**
1)Control tab:
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard->click on the "control tab " in the left nav bar-> navigate to the "Metricsdc" tab.
 
![image](https://user-images.githubusercontent.com/54656883/66652851-52ad2e00-ec54-11e9-8a35-eb1710b0fe8a.png)

 
**Expected:**
When screen reader users navigate to the  "SQL Servers ","HDFS", "Sparks", "Control", "Gateway" and "App" controls, they should be announced with proper role as tab list , tab, and tab panel and even the selection of the currently selected tab should be announced to the users.

**User Impact:**
If proper role and proper structure of an interactive element is not announced to the screen reader users then they will get confused and might not know how to interact with which can lead to miss out some important information.

**Attachment for Reference:**
[7666_A11y_AzureDataStudio_BigDataClusterOverviewPage_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3717107/7666_A11y_AzureDataStudio_BigDataClusterOverviewPage_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![7666_Closed](https://user-images.githubusercontent.com/54656883/72725343-87ae1280-3bab-11ea-84e0-21be047978f3.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
